src/components/RegisterForm/RegisterForm.tsx

```diff
@@ -0,0 +1,109 @@
+import { ErrorMessage, Field, Form, Formik, FormikHelpers } from 'formik';
+import css from './AuthForm.module.css';
+import { registerSchema } from '../../services/schemas';
+import { FcGoogle } from 'react-icons/fc';
+
+const RegisterForm: React.FC = () => {
+  type FormData = {
+    name: string;
+    tel: string;
```

Comment:
Please, try to use full name for variables, don't shrink it in future
```suggestion
    phone: string;
```

Reply:
done